en/02_Developer_Guides/03_Forms/Field_types/04_GridField.md

```diff
@@ -182,11 +189,10 @@ developers manually adding each component.
 
 A simple read-only and paginated view of records with sortable and searchable headers.
 
-
 ```php
 use SilverStripe\Forms\GridField\GridFieldConfig_Base;
 
-$config = GridFieldConfig_Base::create();
+$config = GridFieldConfig_Base::create(itemsPerPage: 10);
```

Comment:
Are we comfortable using named params in docs? Is it officially support now?

Reply:
Oh, oops. Yes, but not in `create()`, won't work there.

Reply:
Okay. Correction - because that's the only argument for that constructor it _would_ have worked, but only coincidentally. So still removed it.